applications/crossbar/src/modules/cb_clicktocall.erl

```diff
@@ -447,6 +449,7 @@ build_originate_req(Contact, Context) ->
        ,{<<"Outbound-Callee-ID-Number">>, CalleeNumber}
        ,{<<"Outbound-Caller-ID-Name">>, FriendlyName}
        ,{<<"Outbound-Caller-ID-Number">>, OutboundNumber}
+	   ,{<<"Outbound-Call-ID">>, CallId}
```

Comment:
@emnvn As @k-anderson says, have a look at cb_modules_util in #1233. It sets a per-endpoint outbound call id